_data/projects/masters-chance-of-admit.yml

```diff
@@ -0,0 +1,12 @@
+name: Master - Chance of Admit
+desc: A website that provides analytics on how different features contribute to your chances of getting into a university of your choice.
+site: https://github.com/BBloggsbott/masters-chance-of-admit
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- html
+- machine-learning
+- data-viz
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/BBloggsbott/masters-chance-of-admit/issues
```

Comment:
@BBloggsbott Please change this to https://github.com/BBloggsbott/masters-chance-of-admit/labels/good%20first%20issue

Reply:
The requested changes have been made

Reply:
Sorry I gave the wrong URL, your tag is "up-for-grabs" so please change the link to https://github.com/BBloggsbott/masters-chance-of-admit/labels/up-for-grabs

Reply:
@BBloggsbott ^^

Reply:
Gotcha